proto/src/google/ground/v1beta1/survey.proto

```diff
@@ -53,3 +65,15 @@ enum Role {
   // User may also manage the survey, update locations of interest, etc.
   SURVEY_ORGANIZER = 3;
 }
+
+// Defines the type of data sharing consent agreement the user can see.
+enum DataSharingConsentType {
```

Comment:
For consistency, we can move this inside `DataShareConsent` above the field where it's used and rename it to `Type`.